tests/unit/Domain/Entity/Entity/ManageEntityTest.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/**
+ * Copyright 2023 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\ServiceProviderDashboard\Tests\Unit\Domain\Entity\Entity;
+
+use PHPUnit\Framework\TestCase;
+use Surfnet\ServiceProviderDashboard\Domain\Entity\Constants;
+use Surfnet\ServiceProviderDashboard\Domain\Entity\ManageEntity;
+
+class ManageEntityTest extends TestCase
```

Comment:
The ManageEntityTest already exists: `sp-dashboard/tests/unit/Infrastructure/Manage/Dto/ManageEntityTest.php`

Please add your test coverage there. If you want a separate test file for your test coverage, that is OK too. But in that case please use the correct namespace & test folder.